$NetBSD: patch-ac,v 1.1 2003/02/06 10:54:18 dmcmahill Exp $

work around the fact that some systems like i386 report either
netbsd (a.out) or netbsdelf (elf) while others like alpha report
netbsd (elf).  This was making alpha be set incorrectly to a.out

--- target.make.orig	Fri Nov 15 15:14:59 2002
+++ target.make	Thu Feb  6 05:45:23 2003
@@ -720,7 +720,7 @@
 ####################################################
 #
-# NetBSD
+# NetBSD a.out
 #
-ifeq ($(findstring netbsd, $(GNUSTEP_TARGET_OS)), netbsd)
+ifeq (@object_fmt@,netbsd-a.out)
 # This is disabled temporarily, because I don't know exactly how
 # to link shared libs. Everything seems to link correctly now but
@@ -754,5 +754,5 @@
 # NetBSD ELF
 #
-ifeq ($(findstring netbsdelf, $(GNUSTEP_TARGET_OS)), netbsdelf)
+ifeq (@object_fmt@,netbsd-ELF)
 HAVE_SHARED_LIBS    = yes
 SHARED_LD_POSTFLAGS = -Wl,-R/usr/pkg/lib -L/usr/pkg/lib
